services/base-service/spa/client/public/elements/pages/person/app-person-works-list-edit.js

```diff
@@ -137,6 +137,17 @@ export default class AppPersonWorksEdit extends Mixin(LitElement)
         lastPrintedYear = newIssueDate;
       }
     });
+
+    // update doi links to be anchor tags
+    this.citations.forEach(cite => {
+      if( cite.DOI && cite.apa ) {
+        // https://doi.org/10.3389/fvets.2023.1132810</div>\n</div>
+        cite.apa = cite.apa.split(`https://doi.org/${cite.DOI}`)[0]
```

Comment:
Do we want DOI here or *Get it at UC* ? Ala
https://search.library.ucdavis.edu/discovery/openurl?institution=01UCD_INST&vid=01UCD_INST:UCD&id=doi:10.1016%2Fj.compag.2018.09.042

OR both, eg. https://rc.experts.ucdavis.edu/work/2364120

Reply:
@Vensberg 

Could you see @qjhart comment? Essentially in the citations I made the existing `https://doi.org/..` links hyperlinks for each citation, after reading your [comment here](https://github.com/ucd-library/aggie-experts/issues/169#issuecomment-1751409952).

We could also link to our library search, or both.

Reply:
@UcDust let's keep the DOI link because that is often expected. When we do work landing pages for 2.1, those will contain Get it at UC. I think at that time we'll unlink the dois, but tbd closer to that time.